### PR TITLE
Packaging: use permanent selinux label on logs

### PR DESCRIPTION
**Pull request by mnecas**
@mwperina @dangel101

### CODE REVIEW

**Comment by mnecas** [comment]
-1 please don't merge yet

File: misc/packaging/apache/ansible-runner-service.spec [hunk 1]
```diff
@@ -66,7 +66,8 @@ install -m 644 ./misc/packaging/logrotate/ansible-runner-service %{buildroot}%{_
 install -m 644 ./ansible_runner_service.py %{buildroot}%{python3_sitelib}/runner_service
 
 %post
-[[ -f /var/log/ovirt-engine/ansible-runner-service.log ]] && chcon -t httpd_log_t /var/log/ovirt-engine/ansible-runner-service.log
+semanage fcontext -a -t httpd_log_t /var/log/ovirt-engine/ansible-runner-service.log 2> /dev/null || semanage fcontext -m -t httpd_log_t /var/log/ovirt-engine/ansible-runner-service.log
```
Comment (mwperina) [inline misc/packaging/apache/ansible-runner-service.spec:69]: Shouldn't we also check if /var/log/ovirt-engine/ansible-runner-service.log exists?

Comment (mnecas) [reply misc/packaging/apache/ansible-runner-service.spec:69]: This only adds the tag to the database or updates it if there was previously added.

**Comment by mnecas** [comment]
I have done some tests and everything looked fine.

**Comment by mnecas** [comment]
I have done some tests and looked like no issues.

**Review by mwperina — APPROVED** [review]
+1

**Comment by michalskrivanek** [comment]
LGTM

**Pull request merged**